ExerciseOne/exercise-one.html

```diff
@@ -1,160 +1,102 @@
-    <script type="text/javascript">
-        console.log("javascript code goes here")
-    </script>
+    <title>Nike Store Tab Title</title>
+    <style>
+        .logo {
+            display: block;
+            margin: 0 auto;
+            width: 100px;
+            }
+        nav {
+            background: lightblue;
+            }
+            ul.nav-links {
+                display: block;
+                width: 500px;
+                margin: 0 auto;
+                list-style: none;
+                padding: 0px;
+                text-align: center;
+            }
+            ul.nav-links > li {
+                list-style: none;
+                display: inline-block;
+                padding-left: 15px;
+                padding-right: 15px;
+            }
+            ul.nav-links > li > a.link {
+                color: red;
+            }
+            ul.nav-links > li > a {
+                color: orange;
+            }
+            h1.header-title {
+                color: green;
+                font-size: 30px;
+            }
+            .main-section {
+                display: block;
+                padding-top: 50px;
+                margin: 0 auto;
+                width: 800px;
+            }
+            .main-section > section, .main-section > aside {
+                display: inline-block;
+                vertical-align: top;
+            }
+            .main-section > section {
+                width: 530px;
+            }
+            .main-section > aside {
+                width: 240px;
+            }
+            footer {
+                margin-top: 50px;
+                margin-bottom: 50px;
+                text-align: center;
+            }
+            footer p {
+                margin-top: 30px;
+                color: blue;
+            }
+    </style>
 </head>
-    <body>
-        
-        <title>Nike Store tab title
-
-
-
-
-
-        
-
-        <div class="container">
-
-
-
-
-
-<header>
-    <img class="logo" src="https://img.etimg.com/thumb/msid-59738997,width-640,resizemode-4,imgsize-21421/nike.jpg" />
-
-
-
-
-
-
-
-            
+<body>
+    <nav>
+        <ul class="nav-links">
+            <li><a class="link" href="index.html">Home</a></li>
+            <a class="link" href="about.html">About</a></li>
+            <li><a class="link" href="about.html">Shoes</a></li>
+            <li><a class="link" href="about.html">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="container">
+        <header>
+            <img class="logo" src="https://img.etimg.com/thumb/msid-59738997,width-640,resizemode-4,imgsize-21421/nike.jpg" />
             <section class="main-section">
                 <aside class="ads">
                     <img class="aside-pic" src="https://lh3.googleusercontent.com/proxy/kwRV4L36eJUkkQl7dNiXpGzMEmvzLR7u1RwodkhSu47ZeHeimI322Drry4NaVe3ZWoj5Op9LQswnN0Qejhbt8TFjTrw5aXykvGv9-mB7h-CRAiamOrXyGWVtv1juNfz2Frgmc7wGj1DmLeijUEBaAhPGSSFfgRB1u0U" />
                 </aside>
```

Comment:
This `<aside>` and its children do not belong here.  You need to move this to the correct position in this document, based off of the image.